client-app/ui-kit/components/atoms/typography/vc-typography.types.d.ts

```diff
@@ -0,0 +1,7 @@
+declare global {
+  type VcTypographyVariantType = "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "base";
+  type VcTypographyWeightType = "thin" | "light" | "normal" | "bold" | "black";
+  type VcTypographyTextTransformType = "uppercase" | "lowercase" | "capitalize" | "none";
+}
+
+export {};
```

Comment:
You was needed to add `export {}` to tell Typescript it's a module, if there is not `import`